cmd/ocm/create/cluster/cmd.go

```diff
@@ -54,18 +57,21 @@ var args struct {
 	podCIDR     net.IPNet
 }
 
+const clusterNameHelp = "The specified name is used as a DNS component, as well as initial display_name."
```

Comment:
This will be used when generating a sub-domain for your cluster on openshiftapps.com.